### ISSUE TITLE
Container becomes unresponsive/hangs after left idle

### ISSUE THREAD

**Issue by philipzae** [schuellerf/xfce-test#47]
If the window is left idle for some time, when I switch back to it, its not longer responsive and then I have to close the window and restart xfce-test.

**Comment by schuellerf** [comment]
I also saw this once - but as you described, we could double check if this might be related to screensaver or powermanager idle settings!?

**Comment by philipzae** [comment]
This happens to me everytime I use it and leave it idle for a bit of time. :smile:

**Comment by schuellerf** [comment]
I just checked the default settings:
Screensaver: 10min.
Powermanager screen blank: 10min.
Powermanager screen off: 15min.
you could tweak those settings (to different values to be able to distinguish) and see if one of those values impact your problem

**Comment by schuellerf** [comment]
I now tried this and I can reproduce. When you activate the screensaver `xfce4-screensaver-command --activate` it's non responsive
If you use the local terminal (where you started the container) and run `xfce4-screensaver-command --deactivate` manually everything is fine again.
Please confirm, then I would rephrase this issue with "xfce4-screensaver problem" to be fixed.
As a workaround you could increase the time or disable the screensaver.

**Comment by philipzae** [comment]
I turned of the screensaver through the screensaver settings and also tried the `--deactivate` option and both didn't work for me.

**Comment by schuellerf** [comment]
Please also try to disable the power management settings especially those which are screen related.

**Comment by philipzae** [comment]
Tried it and still hung.

**Comment by smac89** [comment]
Hey guys I faced this [issue](https://github.com/mviereck/x11docker/issues/345) as well when creating the [archlinux container](https://github.com/xfce-test/container-archlinux) for xfce. 

The solution is to copy these [files](https://github.com/xfce-test/container-archlinux/tree/main/container/etc/xdg/xfce4/xfconf/xfce-perchannel-xml) into `/etc/xdg/xfce4/xfconf/xfce-perchannel-xml` folder.

It's more of a hack really because as many of you have rightly discovered, turning the screensaver off does not solve the problem, so in addition to that, we also tell the power manager to go into presentation mode from the start. This creates an inhibitor that disables most (all?) powersaving features.

I also reported the issue on [gitlab](https://gitlab.xfce.org/xfce/xfce4-power-manager/-/issues/132).

**Comment by smac89** [comment]
From the [comment](https://gitlab.xfce.org/xfce/xfce4-power-manager/-/issues/132#note_30900) given by @ochosi on gitlab, it looks like we can turn off display power management from the xserver by passing the option: `-dpms`. This option will turn off dpms and from a short test, it behaves the same as enabling presentation mode
